src/main/java/com/uid2/admin/vertx/service/OperatorKeyService.java

```diff
@@ -309,6 +321,45 @@ private void handleOperatorDisable(RoutingContext rc, boolean disableFlag) {
         }
     }
 
+    private void handleOperatorUpdatePublicPrivateType(RoutingContext rc) {
+        try {
+            // refresh manually
+            operatorKeyProvider.loadContent(operatorKeyProvider.getMetadata());
+
+            final String name = rc.queryParam("name").get(0);
+            Optional<OperatorKey> existingOperator = this.operatorKeyProvider.getAll()
+                    .stream().filter(o -> o.getName().equals(name))
+                    .findFirst();
+            if (!existingOperator.isPresent()) {
+                ResponseUtil.error(rc, 404, "operator name not found");
+                return;
+            }
+
+            List<OperatorKey> operators = this.operatorKeyProvider.getAll()
+                    .stream().sorted((a, b) -> (int) (a.getCreated() - b.getCreated()))
+                    .collect(Collectors.toList());
+
+            boolean isPublicOperator = rc.queryParam("isPublicOperator").get(0) == null ? false :
+                    rc.queryParam("isPublicOperator").get(0).equals("true") ? true : false;
+
+            OperatorKey c = existingOperator.get();
```

Comment:
Don't thing we should name variables with one letter outside of perhaps a one line lambda param. Not sure why it's `c` actually.

Reply:
it was my lazy copy&paste off old code - i think once upon a time there was just ClientKey hence it was 'c' and at some point OperatorKey was created afterwards...